tests/integration/test_interface.py

```diff
@@ -430,41 +409,47 @@ def test_generate_dataset_with_state_filtered(dataset_name: str, request, mocker
         DATASETS.cps.name,
         DATASETS.tax_w2_1099.name,
         DATASETS.wic.name,
-        "TODO: tax_1040",
+        DATASETS.tax_1040.name,
     ],
 )
-def test_generate_dataset_with_state_unfiltered(dataset_name: str, request, mocker, tmpdir):
+def test_generate_dataset_with_state_unfiltered(dataset_name: str, request, mocker):
     """Test that values returned by dataset generators are for all locations if state unspecified"""
     if "TODO" in dataset_name:
         pytest.skip(reason=dataset_name)
     dataset = DATASETS.get_dataset(dataset_name)
-    data = request.getfixturevalue(f"sample_data_{dataset_name}_state_edit")
-    _generate_non_sample_data_root(dataset_name, tmpdir, data)
 
     # Skip noising (noising can incorrect select another state)
     mocker.patch("pseudopeople.interface.noise_dataset", side_effect=_mock_noise_dataset)
-
     noising_function = DATASET_GENERATION_FUNCS[dataset_name]
-    noised_data = noising_function(source=Path(tmpdir))
+    noised_data = noising_function(
+        source=request.getfixturevalue("split_sample_data_dir_state_edit")
```

Comment:
I think you can get this by passing the fixture directly as an argument of the test function rather than getting it off the `request` since it's not a parameterized fixture. If so, check in the subsequent tests. There are many analogous calls.